src/main/webapp/WEB-INF/views/admin/system/configuration-js.jsp

```diff
@@ -56,7 +56,8 @@
 				
 				var loginDetail = {};
 			<c:forEach var="code" items="${ct:getCodeValues(ct:getConstDef('CD_LOGIN_SETTING'))}" varStatus="status">
-				loginDetail["${code[0]}"] = $("#ldap${code[0]}").val();
+				loginDetail["${code[0]}"] =
+                    $("#ldap${code[0]}").prop("tagName") === "SELECT" ? $("#ldap${code[0]} option:selected").val() : $("#ldap${code[0]}").val();
```

Comment:
@acisliver How about changing it like below? It would be more readable.

```js
var isSelected = $("#ldap${code[0]}").prop("tagName") === "SELECT";
loginDetail["${code[0]}"] = $("#ldap${code[0]}" + isSelected ? " option:selected" : "").val());
```